### DISCUSSION TITLE
A bit lost with seaport and openSea js

### DISCUSSION THREAD

**Discussion by FloUz-ai** [ProjectOpenSea/seaport-js#103] [Q&A] [answered]
How can we use seaport to buy and sell items on opensea ? 

I went through the doc twice but honnestly it is a bit hard to understand

**Comment by bbk1985** [comment]
did you find any guides in the end? i'm trying this out and it's quite difficult for me

**Comment by FloUz-ai** [comment]
Not really, but I managed to make it work by going through some others tickets and trying to reproduce the order than u can see on opensea

**Comment by ryanio** [comment, marked as answer]
For an example check out https://github.com/montasaurus/bidding-bot-demo
